tools/clang/test/HLSLFileCheck/hlsl/payload_qualifier/structs.hlsl

```diff
@@ -0,0 +1,28 @@
+// RUN: %dxc -T lib_6_6 -validator-version 1.6 %s -allow-payload-qualifiers | FileCheck %s
```

Comment:
lib_6_6 will already constrain validator version, so no need for that argument.

Reply:
removed